Makefile

```diff
@@ -25,7 +25,7 @@
 # Current director
 PROJECT_DIR=$(shell pwd)
 
-BUILD_NUMBER=2
+BUILD_NUMBER=3
```

Comment:
For future reference - I'll bump this on in preparation for a new build; you don't have to include the change in your patch.